### PR TITLE
Fix: [IGL-106] Input character count color fix and version updates

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

**Review by fraincs — APPROVED** [review]

File: packages/Disclosure/src/disclosure.scss [hunk 1]
```diff
@@ -142,6 +142,10 @@
 
 .ids-disclosure .ids-disclosure__header-icon {
     margin: var(--ids-disclosure-header-icon-margin);
+
```
Comment (alexasselin008) [inline packages/Disclosure/src/disclosure.scss:145]: This change seems undocumented

File: .changeset/clean-needles-taste.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+"@igloo-ui/text-editor": patch
+"@igloo-ui/textarea": patch
+"@igloo-ui/toaster": patch
+"@igloo-ui/toggle": patch
+"@igloo-ui/tooltip": patch
+"@igloo-ui/vertical-bar-chart": patch
+"docs": patch
+---
+
+Updated input icon and character count colors. Also updated hopper token versions and igloo icon versions.
```
Comment (alexasselin008) [inline .changeset/clean-needles-taste.md:51]: i think the best practice here would be to create 2 changeset

texteditor with the message: Updated input icon and character count colors
all other pacakges: updated hopper token versions and igloo icon versions.

File: packages/Disclosure/src/disclosure.scss (hunk 1 above)
Comment (vicky-comeau) [reply packages/Disclosure/src/disclosure.scss:145]: ya when I was testing I noticed that the variable wasn't written right.. should I add this to my changeset?

File: .changeset/clean-needles-taste.md (hunk 2 above)
Comment (vicky-comeau) [reply .changeset/clean-needles-taste.md:51]: ok I will created two changesets, thanks

File: packages/Disclosure/src/disclosure.scss (hunk 1 above)
Comment (vicky-comeau) [reply packages/Disclosure/src/disclosure.scss:145]: I updated the changesets

**Review by alexasselin008 — APPROVED** [review]

**Review by fraincs — APPROVED** [review]

**Pull request merged**